src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const discount = calculateDiscount(days);
+  const finalPrice = days * 40;
```

Comment:
The calculation of `finalPrice` should include the discount. Currently, it calculates the total price without considering the discount, which is subtracted later. Consider calculating the total price after applying the discount directly.